backend/supabase/love/love-compatibility-answers.sql

```diff
@@ -0,0 +1,41 @@
+create table if not exists
+  love_compatibility_answers (
+    id bigint generated always as identity primary key,
+    question_id bigint not null,
+    creator_id text not null,
```

Comment:
This is the user who answered it right, not the creator of the question? I guess it's fine, but might have been better as user_id?